### ISSUE TITLE
Allow setting GL_ARB_arrays_of_arrays

### ISSUE THREAD

**Issue by haasn** [KhronosGroup/glslang#1057]
from glslangValidator master:
```
WARNING: /tmp/ravk.glsl:3: '#extension' : extension not supported: GL_ARB_arrays_of_arrays
```

https://www.khronos.org/registry/OpenGL/extensions/ARB/ARB_arrays_of_arrays.txt

**Comment by haasn** [comment]
I'd like to point out that it seems glslang actually *does* support nested arrays, it just doesn't advertise support for the extension. That is, the `#extension` pragma gets rejected, but just omitting the extension line makes it work fine.

**Comment by haasn** [comment]
GL_ARB_shader_storage_buffer_object is also rejected even though it should work fine.

**Comment by greg-lunarg** [comment]
Closed as duplicate to #2649
